plugins/nlp/nlp_eng.hpp

```diff
@@ -0,0 +1,36 @@
+/*##############################################################################
+1
+    HPCC SYSTEMS software Copyright (C) 2021 HPCC Systems�.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//#define NLP_DEBUG
+
+#ifndef _WIN32
+#define LINUX 1
```

Comment:
This seems pointless...

Reply:
This code can  be removed now the jlib function is being used.

Reply:
Why resolve when you haven't removed the code?